examples/stm32f4_discovery/sab2/main.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2018, Carl Treudler
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#include <modm/board.hpp>
+#include <modm/communication/sab.hpp>
+#include <modm/communication/sab2.hpp>
+
+// ----------------------------------------------------------------------------
+//
+// NOTE: This example requires a connection between PB6 and PC11.
+//
+
+modm::sab2::Interface<Usart1, 20> sabA;
+modm::sab2::Interface<Uart4, 20> sabB;
+
+int
+main()
+{
+	Board::initialize();
+	Board::LedBlue::set();
+
+	Usart1::connect<GpioB6::Tx>();
```

Comment:
You can connect Tx and Rx in a single line:
```cpp
Usart1::connect<GpioB6::Tx, GpioB7::Rx>(); 
```

Reply:
Not just *can*, you *should*, so that the API can check for pin group remap violations.